libc/test/src/math/smoke/nanf128_test.cpp

```diff
@@ -54,8 +54,9 @@ TEST_F(LlvmLibcNanf128Test, RandomString) {
 }
 
 #if !defined(LIBC_HAVE_ADDRESS_SANITIZER) && defined(LIBC_TARGET_OS_IS_LINUX)
-#include <signal.h>
-TEST_F(LlvmLibcNanf128Test, InvalidInput) {
-  EXPECT_DEATH([] { LIBC_NAMESPACE::nanf128(nullptr); }, WITH_SIGNAL(SIGSEGV));
-}
+// #include <signal.h>
```

Comment:
Can we use `UNIT_TEST_ONLY` in the CMake?

Reply:
These changes to nan tests are just temporary and will be removed in the following updates.  Their proper fix is in https://github.com/llvm/llvm-project/pull/106468